zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGeneratorTest.scala

```diff
@@ -530,6 +530,123 @@ class AQLGeneratorTest extends Specification {
           |}""".stripMargin.trim)
     }
 
+    "translate lookup one table with one join key" in {
+      val populationDataSet = PopulationDataStore.DatasetName
+      val populationSchema = PopulationDataStore.PopulationSchema
+
+      val selectStatement = SelectStatement(Seq.empty, 0, 0, Seq("*", "population"))
+      val lookup = LookupStatement(Seq("geo_tag.stateID"), populationDataSet, Seq("stateId"), Seq("population"),
+        Seq("population"))
```

Comment:
for the multiple parameters case, 
1. usually we want a better alignment. 
2. we can explicitly write the parameter's name to make the unamed varaibles meaningful. e.g., you can write
```
val lookup = LookupStatement(
sourceKeys = Seq("geo_tag.stateID"), 
dataset = populationDataSet, 
lookupKeys = Seq("stateId"), 
selectValues= Seq("population"),
as = Seq("population"))
```